drf_multitokenauth/views.py

```diff
@@ -75,6 +72,7 @@ def post(self, request, *args, **kwargs):
                 user=user,
                 user_agent=request.META.get('HTTP_USER_AGENT', ''),
                 last_known_ip=get_client_ip(request)[0],
+                token_name=token_name,
```

Comment:
We should also add a block in the readme where we describe that the name is taken form the request and used on create,

Reply:
I simply added it in the `How to use` part, where the login is described - is that sufficient or would you prefer an example request?

Reply:
looks good for now, we just need to make sure that the feature is visible in the docs :wink: